net-dialup/lrzsz/lrzsz-0.12.20.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="Communication package providing the XMODEM, YMODEM & ZMODEM protocols"
+DESCRIPTION="lrzsz is a Unix package providing the XMODEM, YMODEM AND ZMODEM \
+file transfer protocols."
+HOMEPAGE="https://ohse.de/uwe/software/lrzsz.html"
+COPYRIGHT="1996-1998 Uwe Ohse"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://ohse.de/uwe/releases/lrzsz-$portVersion.tar.gz"
+CHECKSUM_SHA256="c28b36b14bddb014d9e9c97c52459852f97bd405f89113f30bee45ed92728ff1"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	lrzsz = $portVersion
+	cmd:lrz = $portVersion
+	cmd:lsz = $portVersion
+	cmd:lrb = $portVersion
+	cmd:lrx = $portVersion
+	cmd:lsb = $portVersion
+	cmd:lsx = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:ld
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i -e "s|ln|ln -r -s|" src/Makefile.in
+}
+
+BUILD()
+{
+	runConfigure --omit-dirs "docDir dataRootDir" ./configure
+	make $jobArgs LIBS="-lnetwork"
+}
+
+INSTALL()
+{
+	make install
+	mkdir -p $dataDir
+	mv $prefix/share/* $dataDir
+	rmdir $prefix/share/
```

Comment:
The trailing slash in "share/" does not harm, but it is not needed either, and the reader already knows it is a directory, since the command we're calling is a simple rmdir.
So imho it would be better to drop that trailing slash.
(That said, you can keep it if you wish ;-)
Hmm, maybe you did not notice I had dropped it, or maybe you forgot to remove it.
Anyway, you can do as you wish ;-)